Cargo.toml

```diff
@@ -16,8 +16,10 @@ tokio-process = "0.1.5"
 tokio-timer = "0.1"
 
 clap = "2.20.0"
-semver = "0.5.1"
+semver = "0.9.0"
```

Comment:
Good idea to keep these things up to date.

Reply:
Yes, and it's a requirement for `Eq` and `Hash`, see 2e8885c9c6d2c0e69a07a3faef2b886893ed50e8.

Reply:
Ah good catch!